index.html

```diff
@@ -9,13 +9,25 @@
     <link rel="stylesheet" href="styles.css">
   </head>
   <body>
-  
-    <!-- START HEADER -->
+   <header>
 
-    <!-- END HEADER -->
+    <h1>Juan Fabrega</h1> <h3>posts</h3><h3>about</h3>
+    &nbsp;
+   </header>
+   <h4>I am a software engineer with a passion for craftmanship</h4><br>
+    <br>
+    <h4>My Recent Posts</h4>
+   <ol>
+      <li><a href="https://google.com">My First Post</a></li>
```

Comment:
Remember these posts need to have a title and a description. The anchor tag wrapping the title is good, but I think they should probably be h3 or h4 and then a p tag for the description. Maybe something like this:

```
<li>
  <a>
    <h3>title</h3>
  </a>
  <p>description</p>
</li>
```